en/docs/setup/set-passwords-using-environment-variables-or-system-properties.md

```diff
@@ -20,3 +23,25 @@ key_password = "$env{ENV_VAR}"
 [truststore]                  
 password = "$env{ENV_VAR}" 
 ``` 
+
+### Set passwords using System Properties
+ 1.  Open `<IS_HOME>/repository/deployment.toml` file and refer the required password value in the configuration using `$sys{system.property}` placeholder. 
+   
+    ``` tab="Format"
+    [super_admin]
+    username="admin"
+    password="$sys{system.property}"
+    ```
+        
+    ``` tab="Example"
+    [super_admin]
+    username="admin"
+    password="$sys{admin.password}"
+    ```
+    
+2.  Pass the above configured system property to runtime by using one of following options (`admin.password` has been used as the sample system property)
```

Comment:
2.  Pass the above configured system property to the runtime by using one of following options during server startup. 

      !!! info
          Note that `admin.password` has been used as the sample system property.

       * On **Linux**: `./wso2server.sh -Dadmin.password=admin`
       * On **Windows**: `./wso2server.bat -Dadmin.password=admin`